ocean_lib/web3_internal/web3_overrides/signature.py

```diff
@@ -19,7 +20,12 @@ class SignatureFix(Signature):
     v value of 27 or 28 instead of 0 or 1
     """
 
-    def __init__(self, signature_bytes=None, vrs=None, backend=None) -> None:
+    def __init__(
+        self,
+        signature_bytes: Optional[bytes] = None,
+        vrs: Optional[Tuple[int, int, int]] = None,
+        backend: Optional[object] = None,
```

Comment:
Should be `Optional[Union[BaseECCBackend, Type[BaseECCBackend], str, None]]`. Copied from `Signature.__init`.